.circleci/config.yml

```diff
@@ -92,6 +93,13 @@ commands:
                   fi
       - attach_workspace:
           at: .
+      - when:
+          condition:
+            or:
+              - equal: [ "alpha", << parameters.env >> ]
+              - equal: [ "staging", << parameters.env >> ]
+          steps:
+            - run: cp app.test.yaml app.yaml
```

Comment:
Replace `app.yaml` if env is alpha or staging.

Reply:
Instead of having a separate copy of `app.yaml` that could get out-of-date, How about scripting a modification of `app.yaml` in-place? This `sed` command should do the trick:
```
sed -e 's/default_expiration: ".*"/default_expiration: "0m"/' -i .orig app.yaml
```

Reply:
`0m` feels preferable to me over `30s` in case `30s` is long enough for running into the page not loading issue.

Reply:
`sed`is a great suggestion. I'll change.